src/main/java/foundation/privacybydesign/email/EmailRestApi.java

```diff
@@ -125,7 +125,7 @@ public Response sendEmailToken(@FormParam("email") String emailAddress,
                     conf.getServerURL(language) + "#verify-email/" + token
             );
         } catch (AddressException e) {
-            logger.error("Invalid address: {}: {}", emailAddress, e.getMessage());
+            logger.error("Invalid address: {}", e.getMessage());
```

Comment:
The same here.

Reply:
verified: in the message of the exception `emailAddress` is not metioned